### ISSUE TITLE
gettingstarted.md outdated

### ISSUE THREAD

**Issue by Wynadorn** [tiltedphoques/TiltedOnline#134]
Something minor but `gettingstarted.md` is outdated, it mentions a non existing `\Build\` directory , I think new build instructions can now be found here: https://wiki.tiltedphoques.com/tilted-online/coding/build-guide

*side note
>![image](https://user-images.githubusercontent.com/4095127/129488569-0fd9d20c-c8cc-46a2-af16-f5c7548be522.png)
> As mentioned in the Readme you will need Visual Studio 2019 

But the readme no longer mentions this.

**Comment by yamashi** [comment]
You're right, at this point I am not sure if we should leave the build instructions in the readme or just put a link to the wiki, what do you think?

**Comment by Wynadorn** [comment]
Probably best to avoid any duplicate information, just like you would with code.
So definitely remove it. 

Just noticed that `readme.md`'s build guide link already points to the wiki, so removing `gettingstarted.md` should be fine.

Another minor improvement could be made to the wiki page, you can put a url on 
`https://github.com/tiltedphoques/TiltedOnline/wiki`
instead of 
`https://github.com/tiltedphoques/TiltedOnline/wiki/TiltedOnline-Wiki`

example:
https://github.com/runelite/runelite/wiki

**Comment by Dragonisser** [comment]
> Probably best to avoid any duplicate information, just like you would with code.
> So definitely remove it.
> 
> Just noticed that `readme.md`'s build-guide link already points to the wiki, so removing `gettingstarted.md` should be fine.
> 
> Also a small improvement could be made to the github wiki page: You can put a url on:
> `https://github.com/tiltedphoques/TiltedOnline/wiki`
> instead of
> `https://github.com/tiltedphoques/TiltedOnline/wiki/TiltedOnline-Wiki`
> 
> example:
> [https://github.com/runelite/wiki](https://github.com/runelite/runelite/wiki)

Afaik we arent even using the github wiki anymore, should be removed since we have everything here 

https://wiki.tiltedphoques.com/tilted-online/coding/build-guide
